examples/1-basic/galaxy-redshift-distributions.ipynb

```diff
@@ -0,0 +1,645 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "image/png": {
+       "height": 433,
+       "width": 578
+      }
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# show the expected galaxy spectrum from the simulation\n",
+    "ell = np.arange(lmax + 1)\n",
+    "plt.plot(ell[1:], clustering[1:], label=\"clustering\")\n",
+    "plt.plot(ell[1:], lensing[1:], label=\"lensing\")\n",
+    "plt.plot(ell[1:], cross[1:], label=\"cross\")\n",
+    "plt.legend()\n",
+    "plt.xscale(\"log\")\n",
+    "plt.yscale(\"log\")\n",
+    "plt.xlabel(\"$\\\\ell$\")\n",
+    "plt.ylabel(\"$C_\\\\ell$\")\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally, we can compare this with the expected signal from theory **using the original $n(z)$**."
```

Comment:
This is rendering as:
<img width="844" alt="Screenshot 2025-03-17 at 14 23 43" src="https://github.com/user-attachments/assets/5be8493a-36d0-4d62-9235-605efbf716e7" />

note the `:math: n(z)`